### PR TITLE
Session.isOpen method change return type from Boolean to boolean

### PR DESCRIPTION
**Pull request by kbetl-dlb**

### CODE REVIEW

File: src/services/session/SessionService.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export class SessionService {
   /**
    * Returns if a session that connects the SDK with the Dolby.io platform is opened or not.
    */
-  public async isOpen(): Promise<Boolean> {
+  public async isOpen(): Promise<boolean> {
     return this._nativeModule.isOpen();
```
Comment (graduad) [inline src/services/session/SessionService.ts:34]: Does this not need some kind of conversion? Ie. `this._nativeModule.isOpen().map((b) => { return b.valueOf(); });` .

Comment (kbetl-dlb) [reply src/services/session/SessionService.ts:34]: no, we do not need this. Based on this page: https://reactnative.dev/docs/0.62/native-modules-ios#argument-types, boolean is one of types that is automatic converted.

Comment (graduad) [reply src/services/session/SessionService.ts:34]: So are you saying that `this._nativeModule.isOpen()` has always returned a `boolean`? If not, how would it know what to return? Based on the return type of the function? And if so what about plain javascript?

Comment (graduad) [reply src/services/session/SessionService.ts:34]: After trying this in @kbetl-dlb and myself found that `this._nativeModule.isOpen()` a `Promise<boolean>`. Hence this PR is correct.

**Review by graduad — APPROVED** [review]

**Pull request merged**